docker-compose.yml

```diff
@@ -84,8 +84,21 @@ services:
       retries: 10
     depends_on:
       - postgres
+  metadata-service:
+    image: "quay.io/cdis/metadata-service:2021.03"
+    depends_on:
+      - postgres
+    environment:
+      - DB_HOST=postgres
+      - DB_USER=postgres
+      - DB_PASSWORD=postgres
+      - DB_DATABASE=metadata_db
```

Comment:
Is this actually the proper db? 🤔

Reply:
no, in compose-services `metadata_db` is the DB used by sheepdog and peregrine (it's call `sheepdog` in cloud-automation). The metadata-service needs its own DB, which is called `metadata` in cloud-automation.

Not confusing at all right? 😄 you'll need to add the creation of a new DB

Reply:
Having `index_not_found_exception` in the logs is definitely an improvement 😄

Reply:
😄 Thanks! I'll add that